src/javascript/JContent/ContentRoute/ContentLayout/queryHandlers/FilesQueryHandler.js

```diff
@@ -6,7 +6,6 @@ export const FilesQueryHandler = {
 
     getQueryVariables: selection => ({
         ...BaseQueryHandler.getQueryVariables(selection),
-        includeUsageCounts: true,
```

Comment:
this is just leftover from my last pr that I forgot to delete, not used anymore. Deleted here